docs/docs/3-features-modules/10-warning-module/index.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Warning Module
+---
+
+# Warning Module: Overview and Implementation Guide
+
+The Warning Module helps find and report issues in the project.  
+All warnings are shown in **one central place** inside a **grid in the admin panel**.
+
+In this grid, you can:
+
+- See **all warnings that concern you** (meaning you have the necessary scope permissions).
+- Click on an action to **jump directly to the issue** and fix it.
+
+![Warnings Grid](images/warnings-grid.png)
+
+The Warning Module operates through two main mechanisms:
+
+1. **Daily Background Job**  
+   A scheduled job runs once per day to check all warnings across the system.
+
+2. **Real-time Updates**  
+   Whenever a block or entity is modified, the system automatically checks warnings for that specific item.
+
+This dual approach ensures both thorough checks through daily scans and immediate feedback when changes are made.
+
+## Implementation Guide
+
+New projects come with the Warning Module **already set up** by default.  
+However, if your project does **not** have the Warning Module configured yet, you need to **register it manually**.
+
+### 1. Register the Warning Module for the API in `app.module.ts`
+
+Add the module to the `imports` array:
+
+```typescript
+import { WarningsModule } from "@comet/cms-api";
+
+return {
+    module: AppModule,
+    imports: [
+        // other modules...
+        WarningsModule,
+    ],
+};
+```
+
+### 2. Add the Warnings Page to the Admin Menu
+
+To make the warnings accessible in the admin panel, add the following entry to the masterMenu:
+
+```typescript
+{
+    type: "route",
+    primary: <FormattedMessage id="menu.warnings" defaultMessage="Warnings" />,
+    route: {
+        path: "/system/warnings",
+        component: WarningsPage,
+    },
+    requiredPermission: "warnings",
+}
+```
+
+### 3. Schedule the Daily Warning Check
+
+To ensure warnings are checked regularly, schedule the following CLI command to run once per day:
+
+```
+npm run console check-warnings
+```
+
+It's recommended to run this job during off-peak hours (such as at night) to minimize impact on application performance.
```

Comment:
We could link to the cron jobs docs here.

Reply:
done here: 932b6c15205b577d3ce83b5b94c24015469d413e

Reply:
https://github.com/user-attachments/assets/03bd6e46-6073-4b0c-8fb6-fe358b4bdbe8